packages/runtime/src/traits/core/fetch.tsx

```diff
@@ -1,8 +1,9 @@
-import { useCallback, useEffect, useMemo } from 'react';
 import { createTrait } from '@meta-ui/core';
 import { Static, Type } from '@sinclair/typebox';
 import { TraitImplementation } from '../../registry';
 
+let hasFetched = false;
```

Comment:
Using a module-level variable will cause all the fetch traits to share this state?

Reply:
ok, I will fix it.

Reply:
I find it more complex than I thoughts. The best way to fix this problem is a memo mechanism which can detect change of properties, so that trait can decide whether to resend request. Maybe we can put this feature in next milestore.